data/human/outfits.txt

```diff
@@ -394,6 +394,14 @@ outfit "Nerve Gas"
 	"unplunderable" 1
 	description "This is a highly illegal weapon for hand-to-hand combat when boarding an enemy ship. Because chemical weapons are banned by interstellar law, you can be severely fined for getting caught in possession of nerve gas canisters. But, they do clear out an enemy ship quicker than just about anything else..."
 
+outfit "The old mans sword"
+	category "Hand to Hand"
+	cost 120000
+	thumbnail "outfit/old sword"
+	"capture attack" 3
+	"capture defense" 3
+	"unplunderable" 1
+	description "A sword that has all kind of alien technology implemented in it. You have yet to find out what it can do. Some functionalities are a personall shield and an implemented shooting mechanism."
```

Comment:
wait what? this really is stepping on the toes of my warlord blades xD
What about smth like it cuts trough any materials by heating up? And we imagine this sword is simply Korath in origin?

Reply:
You could tie it in with the warlord blades; their father might have studied a few and built his own, but died before he was able to refine it, or he was found by someone who wanted it and needed to keep it safe.

Reply:
Well the warlord and Unfettered warlord blades, and although some could have gotten lost I feel like this blade has more of a Korath/Sheragi style, which is good.
Also I intend to give the blades better stats than this since other than the license they're not obtainable and are there to make capturing warlord ships harder gameplay wise. (many story reasons too but anyway)

I think I like the Sheragi idea best though. The Hai, having fought them, could have been inspired. This one could just have been built from the remainders of one of those Sheragi blades (that would have been much bigger).

Reply:
I feel like Sheragi swords would be too big for the player to handle.

Reply:
Yeah ofc course they would thats why I said this is made out of the remainders of one